common/tensor_common.c

```diff
@@ -1268,3 +1268,18 @@ gst_tensors_typefind_function (GstTypeFind * tf, gpointer pdata)
         gst_caps_new_simple ("other/tensorsave", NULL, NULL));
   }
 }
+
+/**
+* @brief Copy src meta data to dest structure
+*/
+void
+gst_tensor_meta_copy (GstTensor_TensorsMeta * dest, GstTensor_TensorsMeta src)
```

Comment:
I'd supposed to use:
```
gst_tensor_meta_copy (GstTensor_TensorsMeta * dest, const GstTensor_TensorsMeta * src)
```

1. Let's not copy src (memcpy)
2. Let's contify src so that users of meta_copy may keep read-only data constified.

Reply:
I agree with the upper suggestion. It will be applied.